source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -441,6 +443,50 @@ Example:
 
   <queue_size>16384</queue_size>
 
+.. _reference_agents_disconnection_time:
+
+agents_disconnection_time
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.1.0
+
+This sets the time since the last keepalive of an agent to consider it as disconnected.
+
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| **Default value**       | 20s                                                                                                                               |
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| **Allowed values**      | A positive number that should end with a character indicating a time unit, such as: s (seconds), m (minutes), h (hours), d (days) |
++-------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+
+Example:
+
+.. code-block:: xml
+
+  <agents_disconnection_time>1m</agents_disconnection_time>
+
+.. _reference_agents_disconnection_alert_time:
+
+agents_disconnection_alert_time
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.1.0
+
+This sets the time since an agent is considered disconnected to trigger an alert.
```

Comment:
Let's reword a little bit this sentence...

```
This sets the time after which an alert is generated since an agent was considered as disconnected.
```

Then, in the next line...

```
As this time is after an agent is considered as disconnected, the minimum time frame to produce an alert taking the default values is 2m and 20s.
```

Reply:
Done!